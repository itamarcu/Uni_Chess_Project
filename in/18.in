game_mode 2
start
move <2,E> to <4,E>
move <7,E> to <5,E>
move <2,F> to <4,F>
undo
quit
