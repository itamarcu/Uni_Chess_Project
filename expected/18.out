 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Game mode is set to 2-player
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ M _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
8| R N B Q K B N R |
7| M M M M _ M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ M _ _ _ |
4| _ _ _ _ m m _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ _ m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Undo move for white player: <4,F> -> <2,F>
Undo move for black player: <5,E> -> <7,E>
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (black player):
Exiting...
